### PR TITLE
Updated apm.launch to forward new fcu_protocol parameter

### PR DESCRIPTION
**Pull request by sondreengebraten**
Update to allow for passing through fcu_protocol parameter

### CODE REVIEW

File: mavros/launch/apm.launch [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	<!-- example launch script for ArduPilotMega based FCU's -->
 
 	<arg name="fcu_url" default="/dev/ttyACM0:57600" />
+    <arg name="fcu_protocol" default="v1.0" />
```
Comment (TSC21) [inline mavros/launch/apm.launch]: indentation

File: mavros/launch/apm.launch [hunk 2]
```diff
@@ -14,6 +15,7 @@
 		<arg name="config_yaml" value="$(find mavros)/launch/apm_config.yaml" />
 
 		<arg name="fcu_url" value="$(arg fcu_url)" />
+        <arg name="fcu_protocol" value="$(arg fcu_protocol)" />
```
Comment (TSC21) [inline mavros/launch/apm.launch]: indentation

File: mavros/launch/apm.launch (hunk 1 above)
Comment (TSC21) [reply mavros/launch/apm.launch]: And put this and the bellow line under the `log_output` arg so it matches the PX4 launch file.

**Comment by khancyr** [comment]
this is not good, the default protocol should stay mavlink v2.

**Comment by vooon** [comment]
Will fix manually. Thanks!

**Pull request merged**